solvebio/cli/data.py

```diff
@@ -954,17 +956,38 @@ def ls(args):
     return True
 
 
-def _ls(full_path, recursive=False):
+def _ls(full_path, recursive=False, follow_shortcuts=False):
     files = list(Object.all(glob=full_path, limit=1000))
 
     for file_ in files:
-        print(
-            "{}  {}  {}".format(
-                file_.last_modified, file_.object_type.ljust(8), file_.full_path
+        if follow_shortcuts and file_.is_shortcut:
+            shortcut = file_.full_path
+            try:
+                resolved_file = file_.get_target()
+                print(
+                    "{}  {}  {}  from shortcut: {}".format(
+                        resolved_file.last_modified,
+                        resolved_file.object_type.ljust(8),
+                        resolved_file.full_path.ljust(50),
+                        shortcut
+                    )
+                )
+            except NotFoundError:
+                print(
+                    "shortcut target not found!{}from shortcut: {}".format(
```

Comment:
can we rephrase it to indicate that it may not be accessible to the current user?
`Shortcut {} could not be resolved: the target may have been deleted or you may not have permission to access it`

Reply:
Addressed these comments a while ago - forgot to follow up here in the thread sorry